test/suites/network_zone.sh

```diff
@@ -40,6 +40,10 @@ test_network_zone() {
   # Create zone in project.
   lxc network zone create lxdfoo.example.net --project foo
 
+  # Check listing zones from all projects.
+  lxc network zone list --all-projects -fcsv | grep -q default,lxd.example.net || false
+  lxc network zone list --all-projects -fcsv | grep -q foo,lxdfoo.example.net || false
```

Comment:
Why the `|| false`? This is making it a little confusing to read for me as if the pattern is not found, grep will exit 1. If it is found then the "or" will return anyway. In either case, we don't get to the `false`.

Additionally, can we use `-F` for a fixed string here, and have a space between the format flag and `csv`? Thanks.

Reply:
Also can we remove the `-q` from the grep as I think it's generally preferred that we can see it in test output. Thanks.